app/layout.test.tsx

```diff
@@ -0,0 +1,16 @@
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import RootLayout from './layout'
+import { appLabels } from './constants'
+
+describe('Main page: test cases', () => {
+  test('render the main component', () => {
```

Comment:
test is right but FYI we are using it in our project

Reply:
Okay, I'll start using 'it' going forward.